wip_pacta_data_qa.Rmd

```diff
@@ -0,0 +1,545 @@
+* other steps???
+
+```{r}
+no_ALD_CB <- result_data_qa %>% 
+  filter(asset_type == "Bonds" &
+           has_asset_level_data == F &
+           financial_sector != "Other" &
+           investor_name != "Meta Investor") %>% 
+  group_by(financial_sector) %>% 
+  summarise(total_no_ALD_CB = n(), sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Bonds") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(financial_sector, desc(share_total_value))
+datatable(no_ALD_CB)
+
+no_ALD_CB_detail <- result_data_qa %>% 
+  filter(asset_type == "Bonds" &
+           has_asset_level_data == F &
+           financial_sector != "Other" &
+           investor_name != "Meta Investor") %>% 
+  #select(financial_sector, isin, company_name, market_value, value_usd) %>% 
+  select(financial_sector, company_name, market_value, value_usd) %>% 
+  #group_by(financial_sector, isin, company_name) %>% 
+  group_by(financial_sector, company_name) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Bonds") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(financial_sector, desc(share_total_value))
+datatable(no_ALD_CB_detail)
+
+
+```
+
+## Non-PACTA sector with ALD
+
+### overview Equity holdings, non-PACTA sector but with ALD
+Overview of Holdings in non-PACTA sectors for which we do find ALD.
+
+1. Overview on sector level, including count and share of overall equity value affected
+2. Overview on sector-company level, including overall equity value affected
+
+**Actions:**
+
+* This could potentially indicate a wrong sector mapping. For cases of relevant size, check if the core business is mapped correctly.
+* If not, override the mapping in the "fin_sector_overrides.csv" file in the data folder of the PACTA_analysis repo
+* ...?
+
+```{r}
+non_pacta_ALD_EQ <- result_data_qa %>% 
+  filter(asset_type == "Equity" &
+           has_asset_level_data == T &
+           financial_sector == "Other" &
+           investor_name != "Meta Investor") %>% 
+  group_by(sectors_with_assets) %>% 
+  summarise(total_non_pacta_ALD_EQ = n(), sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Equity") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(sectors_with_assets)
+datatable(non_pacta_ALD_EQ)
+
+non_pacta_ALD_EQ_detail <- result_data_qa %>% 
+  filter(asset_type == "Equity" &
+           has_asset_level_data == T &
+           financial_sector == "Other" &
+           investor_name != "Meta Investor") %>% 
+  select(sectors_with_assets, isin, company_name, market_value, value_usd) %>% 
+  group_by(sectors_with_assets, isin, company_name) %>% 
```

Comment:
If you add in the bloomberg_id/corporate_bond_ticker it will make it easier to make the changes to the fin_sector_overrides directly!